src/router/index.js

```diff
@@ -47,7 +55,7 @@ router.beforeEach(async (to, from, next) => {
     await store.dispatch("auth/fetchUser");
     if (!store.getters["auth/isAuthenticated"]) {
       next({
-        name: "home"
+        name: "unauthorized"
```

Comment:
Authentication and Authorization are different.  When your authentication is expired the user should be prompted to sign in again.

Unless I'm misunderstanding, a user should only be directed to the authorization page when accessing a resource that the user doesn't have permission to see.

Reply:
Made revisions; Non logged in users will be sent back to the homepage but those that aren't superusers will be able to select other pages from the navbar but those that they do not have the appropriate permissions to access will show up as the 'unauthorized' page but not log them out or send them back to the homepage locking them out of the site without having to press the back button.

![image](https://user-images.githubusercontent.com/62891235/93119815-752f9e80-f690-11ea-8422-ce7a4752ffee.png)

Reply:
Second revision: 

Unauthorized pages will no longer show in navbar 

![image](https://user-images.githubusercontent.com/62891235/93121243-84afe700-f692-11ea-9b64-2a81e26974ab.png)